### PR TITLE
fix: update object search

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
This is a fix for 1.0.8.

- [ ] Lots of changes in 1.0.9, maybe fix this in another pr

### CODE REVIEW

File: object-search/app.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def config():
 
 
 @click.command()
-@click.option('--task', '-task', type=click.Choice(['index', 'query'], case_sensitive=False))
+@click.option('--task', '-task', "-t", type=click.Choice(['index', 'query'], case_sensitive=False))
```
Comment (JoanFM) [inline object-search/app.py:35]: please keep single quote

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: object-search/app.py (hunk 1 above)
Comment (Yongxuanzhang) [reply object-search/app.py]: Sure, sorry my bad.

**Review by JoanFM — DISMISSED** [review]
there are a lot of parameters in the yaml files that correspond to old jina versions.

`uses_reducing` is not used anymore, as well as separatrd workspace

**Comment by Yongxuanzhang** [comment]
> there are a lot of parameters in the yaml files that correspond to old jina versions.
> 
> `uses_reducing` is not used anymore, as well as separatrd workspace

Do we have the list of deprecated params?

**Comment by JoanFM** [comment]
> > there are a lot of parameters in the yaml files that correspond to old jina versions.
> > `uses_reducing` is not used anymore, as well as separatrd workspace
> 
> Do we have the list of deprecated params?

Unfortunatey not, I can help u later

**Comment by Yongxuanzhang** [comment]
> > > there are a lot of parameters in the yaml files that correspond to old jina versions.
> > > `uses_reducing` is not used anymore, as well as separatrd workspace
> > 
> > 
> > Do we have the list of deprecated params?
> 
> Unfortunatey not, I can help u later

The YAML has been updated with the help of @JoanFM 👍

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**